rundeckapp/grails-app/migrations/changelog.groovy

```diff
@@ -104,5 +104,6 @@ databaseChangeLog = {
         include file: 'core/Webhook.groovy'
         include file: 'core/ConstraintsIndexesKeys.groovy'
         include file: 'core/Tag-3.4.0.groovy'
+        include file: 'core/WorkflowStep.groovy'
```

Comment:
I think this should be included before the 3.4.0 tag, right @imad6639 ?

Reply:
Depends on the intended release.  If this is for 3.4.0 then before the 3.4.0 tag.  If it's for the 3.4.1 release, then a 3.4.1 tag should be created and it should go between the 3.4.0 tag and the 3.4.1 tag

Reply:
FIxed considering that it will be included in 3.4.0. Please, let me know if it should be only in 3.4.1